src/geojson/geojson.ml

```diff
@@ -126,16 +150,17 @@ module Make (J : Geojson_intf.Json) = struct
 
       let of_json json = parse_by_type json parse_coords typ
 
-      let to_json positions =
+      let to_json (positions, bbx) =
         J.obj
           [
             ("type", J.string typ);
+            ("bbox", Option.(if is_some bbx then J.array J.float (get bbx) else J.null));
```

Comment:
Can we extract this function to i.e. have a 

```ocaml
let bbox_to_json_or_null bbx = if is_some bbx then J.array J.float (get bbx) else J.null
```

and replace all of the duplicated code with a call to that function?